generator/README.md

```diff
@@ -30,13 +30,16 @@ The generator reads in from `generator.yml` and writes to `snmp.yml`.
 
 Additional command are available for debugging, use the `help` command to see them.
 
+If you want to use the downloaded mibs only, avoiding pollution from system
```

Comment:
Best to just change the example, rather than complicating the docs.

Reply:
OK, done that and repushed